docs/naps/0-nap-process.md

```diff
@@ -0,0 +1,272 @@
+---
+Author: "Juan Nunez-Iglesias <mailto:jni@fastmail.com>"
+Created: '2022-03-23'
+Status: Active
+Type: Process
+---
+
+(nap0)=
+
+# NAP 0 — Purpose and Process
+
+## What is a NAP?
+
+NAP stands for Napari Advancement Proposal. A NAP is a design document
+providing information to the community, or describing a new feature for
+napari or its processes or environment. The NAP should provide a rationale
+for the proposed change as well as a concise technical specification, if
+applicable.
+
+We intend NAPs to be the primary mechanisms for proposing major new
+features, for collecting community input on an issue, and for documenting
+the design decisions that have gone into napari. The NAP author is
+responsible for building consensus within the community and documenting
+dissenting opinions.
+
+Because the NAPs are maintained as text files in a versioned repository,
+their revision history is the historical record of the feature proposal
+[^id3].
+
+### Types
+
+There are three kinds of NAPs:
+
+1. A **Standards Track** NAP describes a new feature or implementation
+   for napari.
+2. An **Informational** NAP describes a napari design issue, or provides
+   general guidelines or information to the Python community, but does not
+   propose a new feature. Informational NAPs do not necessarily represent a
+   napari community consensus or recommendation, so users and
+   implementers are free to ignore Informational NAPs.
```

Comment:
It is a bit unclear to me what the motivation is for initiating an "Informational" NAP. Is it to get feedback or consensus on an explanation or best practice (e.g., as a precusor for writing some explanatory documentation)?

I think I am just a little thrown off by the fact that an informational NAP doesn't necessarily reflect community consensus when it seems like the point of the NAP process is to build consensus: "Informational NAPs do not necessarily represent a
   napari community consensus or recommendation, so users and
   implementers are free to ignore Informational NAPs."

Reply:
PEP257 is an informational PEP, for example:

https://peps.python.org/pep-0257/

I must admit that looking at the [list of PEPs by category](https://peps.python.org/pep-0000/#index-by-category), I am underwhelmed by the informational PEPs! 😂

Nevertheless, it might be a good model to document decisions that might have generated a bunch of discussion. For example, maybe our release schedule would be an informational NAP?

Reply:
I too am confused about the purpose of informational NAPs. I guess I could see them as "a place to put important discussions that have not yet reached a consensus but still need to be documented somewhere more useful than the meeting notes repository"?

Reply:
I could see value if the informational NAP flags the lack of consensus intentionally, and if they get triaged to make actionable? Rather than informational though, they sound like opinion statements...

Reply:
I've added some more discussion around informational NAPs. Since they are common practice in the broader Python community I am inclined to leave them. I have described the rationale in e1ab3be5.